src/VisualStudio/IntegrationTest/TestSetup/TestExtensionManager.cs

```diff
@@ -49,10 +44,7 @@ public override bool CanHandleException(object provider, Exception exception)
                 return true;
             }
 
-            public override void HandleException(object provider, Exception exception)
-            {
-                errorHandler.HandleError(provider, exception);
-            }
+            public override void HandleException(object provider, Exception exception) => _errorHandler.HandleError(provider, exception);
```

Comment:
Please move the body to a second line so that it's not pushed so far to the right.